sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/MySampleToolbarAdapter.java

```diff
@@ -89,4 +89,11 @@ public void onClick(View v) {
     public int getItemCount() {
         return mList.size();
     }
+
+    public void swap(ArrayList<String> data) {
+        mList.clear();
+        mList.addAll(data);
+
+        refreshDataSet();
+    }
 }
```

Comment:
This is method is heavily used on all adapters in MVP Google Samples and its used to swap our item list when new data comes from the Presenter/Model. I added this so i could create the adapter on the onCreate Activity method and then later swap the item list to simulate network connection on your sample.